server/src/controllers/Reset/ResetController.js

```diff
@@ -0,0 +1,26 @@
+const { StatusCodes } = require('http-status-codes');
+const { products } = require('../../products.json');
+
+class ResetProductsController {
```

Comment:
move the file inside Products folder

Product / ResetController.js

and class should be named: ProductResetController

Reply:
done